docs/source/dataframe-design.rst

```diff
@@ -69,6 +69,56 @@ This keyword is available on all functions/methods that take user provided
 callables (e.g. ``DataFrame.map_partitions``, ``DataFrame.apply``, etc...), as
 well as many creation functions (e.g. ``dd.from_delayed``).
 
+Categoricals
+------------
+
+Dask dataframe divides `categorical data`_ into two types:
+
+- Known categoricals have the ``categories`` known statically (on the ``_meta``
+  attribute). Each partition **must** have the same categories as found on the
+  ``_meta`` attribute.
+- Unknown categoricals don't known the categories statically, and may have
+  different categories in each partition.  Internally, unknown categoricals are
+  indicated by the presence of ``dd.utils.UNKNOWN_CATEGORIES`` in the
+  categories on the ``_meta`` attribute.  Since most dataframe operations
+  propogate the categories, the known/unknown status should propogate through
+  operations (similar to how ``NaN`` does).
+
+For metadata specified as a description (option 2 above), unknown categoricals
+are created.
+
+Certain operations are only available for known categoricals. For example,
+``df.col.cat.categories`` would only work if ``df.col`` has known categories,
+since the categorical mapping is only known statically on the metadata of known
+categoricals.
+
+The known/unknown status for a categorical column can be found using the
+``known`` property on the categorical accessor:
+
+.. code-block:: python
+
+    >>> ddf.col.cat.known
+    False
```

Comment:
Pandas doesn't have a repr for the accessor, and I don't think we should either. This might be something we could add to the new html repr though, or to `info` (as suggested [here](https://github.com/dask/dask/pull/1877#discussion_r96747392)).